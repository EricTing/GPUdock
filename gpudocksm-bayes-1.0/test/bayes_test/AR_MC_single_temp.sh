#!bin/bash
min_temp=0.0046080000
step=1.1
temp=$min_temp


printf "temperature\t\tAR_MC\n"
for ((i=1; i < 20; i += 1))
do
    printf "%.10f\t\t" "$temp"
    ./bayes_dock -floor_temp ${temp}f -ceiling_temp 0.036f -nt 1 -t 0.01f -r 0.04f -ns 300000 -nc 1 -p 1a07C.pdb -l 1a07C1.sdf -s 1a07C1.ff -id 1a07C1 |grep "AR of MC" |awk '{print $7}'

    temp=$(echo "$temp * $step"|bc)
done
