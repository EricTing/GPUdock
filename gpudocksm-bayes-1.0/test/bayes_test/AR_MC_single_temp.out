temperature		AR_MC
0.0046080000		0.152264
0.0050688000		0.171826
0.0055756800		0.204047
0.0061332480		0.249817
0.0067465728		0.268424
0.0074212300		0.302108
0.0081633530		0.328561
0.0089796883		0.367656
0.0098776571		0.403932
0.0108654228		0.447954
0.0119519650		0.481335
0.0131471615		0.494089
0.0144618776		0.535541
0.0159080653		0.558035
0.0174988718		0.584372
0.0192487589		0.602307
0.0211736347		0.630614
0.0232909981		0.651301
0.0256200979		0.675241
